src/managers/commands.js

```diff
@@ -103,6 +103,7 @@ class CommandManager {
         let command = this.get(base);
 
         if (command) {
+            this.bot.managers.stats.increment(command.info.name);
```

Comment:
I think it would be better to do something like ``` `command.${command.info.name}` ``` instead of just `command.info.name`, so as to prevent any name conflicts.

You'll have to change this in all other places as well, of course.